### PR TITLE
Add old kivy docs

### PR DESCRIPTION
**Pull request by OloFuchs**
Not ready to merge yet, just adding the old files back to the documentation. Added the tutorials folder back, open to suggestions for how the format should be file format of the docs should be stuctured though. If we want to make a folder for kivy and react apps. open to suggestions/ leaving it how it is.

### CODE REVIEW

File: website/docs/tutorials/introduction/00_tutorial_intro.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+brain. [**Amiga-app-template-kivy**](https://github.com/farm-ng/amiga-app-template-kivy/tree/main)
+ includes all the files required to begin developing a custom application.
+
+Part 2 ([**camera-streamer-kivy**](https://github.com/farm-ng/camera-streamer-kivy))
+will walk you through building a simple kivy interface to allow you to visualize the
+ oak camera streams from the brain.
+
+Part 3 ([**virtual-joystick**](https://github.com/farm-ng/virtual-joystick-v2)) will
+walk you through a more complicated kivy interface, sending movement controls and
+customizing app requirements.
```
Comment (Hackerman342) [inline website/docs/tutorials/introduction/00_tutorial_intro.md:21]: This content should instead be in either or both of the examples overview page and/or a (new) tutorials overview page. The intro to a given tutorial should be about it and any prereqs, not future tutorials. In the past, this was on the Examples Index page under a tutorials section.

File: website/docs/tutorials/installation/00_overview.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+id: overview
+title: 00 - Installing Custom to Brain
+---
```
Comment (Hackerman342) [inline website/docs/tutorials/installation/00_overview.md:4]: This should be part of the introduction tutorial. The idea of the introduction tutorial is to be able to create, configure, deploy, and run a custom app that does little to nothing

**Review by Hackerman342 — COMMENTED** [review]
Mostly reviewed for structure since it's unclear if the content is updated and ready for review or not...

File: website/docs/tutorials/installation/00_overview.md (hunk 2 above)
Comment (OloFuchs) [reply website/docs/tutorials/installation/00_overview.md:4]: Sounds good.

**Review by Hackerman342 — APPROVED** [review]
Please commit & push the local changes from our review session before merging. Great work!

**Comment by OloFuchs** [comment]
Wohoooo
Party TIME!!!

**Pull request merged**